etc/doc/tutorial/11.1-MIDI-In.md

```diff
@@ -2,60 +2,58 @@
 
 # MIDI In
 
-In this section we will learn how to connect a MIDI controller to send
-events into Sonic Pi to control our synths and sounds. Go and grab a
-MIDI controller such as a keyboard or control surface and let's get
+Learn how to connect a MIDI controller to send
+events into Sonic Pi to control synths and sounds. Grab a
+MIDI controller, such as a keyboard or control surface and let's get
 physical!
 
 
 ## Connecting a MIDI Controller
 
-In order to get information from an external MIDI device into Sonic Pi
-we first need to connect it to our computer. Typically this will be via
+In order to get info from an external MIDI device into Sonic Pi,
+first it must be connected to our computer. Typically via
 a USB connection, although older equipment will have a 5-pin DIN
 connector for which you'll need hardware support for your computer (for
-example, some sound cards have MIDI DIN connectors). Once you've
-connected your device, launch Sonic Pi and take a look at the IO section
-of the Preferences panel. You should see your device listed there. If
-not, try hitting the 'Reset MIDI' button and see if it appears. If
-you're still not seeing anything, the next thing to try is to consult
-your operating system's MIDI config to see if it sees your
-device. Failing all that, feel free to ask questions in our friendly
-forums: https://in-thread.sonic-pi.net
+example, some sound cards have MIDI DIN connectors). Once your device
+is connected, launch Sonic Pi where your device should be listed in the
+I/O section of the Preferences panel. If not, try hitting the 'Reset MIDI'
+button and see if it appears. If you're still not seeing anything,
+the next thing to try is to consult your operating system's MIDI config
+to see if it sees your device. Failing all that, feel free to ask
+questions in the public chat room: http://gitter.im/samaaron/sonic-pi
```

Comment:
I think the gitter chat is mainly for development chat? The in-thread forums or maybe the GitHub issues page may be more suitable here.

Reply:
Correct. Additionally, the gitter chat room is pretty much obsolete now, and we will always point folks elsewhere.